components/create_swagger_config.py

```diff
@@ -394,6 +394,11 @@ def isi_schema_to_swagger_object(isi_obj_name_space, isi_obj_name,
                 'properties': {'settings': isi_schema}
             }
             log.warning("Found missing event 'settings' property")
+    elif sub_obj_namespace == 'SmbShares' and 'settings' in props:
+        props['shares'] = {
+            'items': props['settings'], 'minItems': 0, 'type': 'array'}
+        del props['settings']
+        log.warning("Found 'shares' mislabeled as 'settings'")
```

Comment:
This isn't something that needs to be addressed now, but I'm noticing that this function is getting pretty big and unwieldy.  I had to expand the diff quite a few times to see where `props` came from. This might be due for some refactoring soon.

Reply:
Yes, the `isi_schema_to_swagger_object` function has certainly become unwieldy with all of it's schema corrections. I will look into refactoring this as a follow up PR.